notifications_utils/recipient_validation/errors.py

```diff
@@ -36,6 +37,7 @@ class Codes(StrEnum):
         ),
         Codes.UNKNOWN_CHARACTER: "Mobile numbers can only include: 0 1 2 3 4 5 6 7 8 9 ( ) + -",
         Codes.UNSUPPORTED_COUNTRY_CODE: "Country code not found - double check the mobile number you entered",
+        Codes.PREMIUM_NUMBER: "This is a premium rate Number - this is not supported",
```

Comment:
worth checking with saima about what the error message should be.

esp if we're changing the validation to not just catch premium rate, i'd start with something like "This phone number is not supported" or "Not a valid landline or mobile" as starter for ten